x-pack/platform/plugins/shared/observability_ai_assistant/server/utils/recall/recall_and_score.ts

```diff
@@ -72,15 +72,15 @@ export async function recallAndScore({
 
     analytics.reportEvent<RecallRanking>(recallRankingEventType, {
       scoredDocuments: suggestions.map((suggestion) => {
-        const llmScore = scores.find((score) => score.id === suggestion.id);
+        const llmScore = llmScores.find((score) => score.id === suggestion.id);
         return {
-          elserScore: suggestion.score ?? -1,
-          llmScore: llmScore ? llmScore.score : -1,
+          elserScore: suggestion.esScore ?? -1,
```

Comment:
esScore instead of `elserScore` for consistency?
```suggestion
          esScore: suggestion.esScore ?? -1,
```

Reply:
Thanks @sorenlouv 
Updated